lib/rubocop/cop/style/redundant_parentheses.rb

```diff
@@ -102,7 +103,7 @@ def check(begin_node)
           return offense(begin_node, 'a variable') if node.variable?
           return offense(begin_node, 'a constant') if node.const_type?
 
-          check_send(begin_node, node) if node.send_type?
+          check_send(begin_node, node) if method_call_type?(node)
```

Comment:
There is `node.call_type?`

Reply:
Thank you for the great suggestion! I updated this PR.